python-avd/pyavd/_schema/models/avd_list.py

```diff
@@ -236,3 +236,17 @@ def _cast_as(self, new_type: type[T_AvdList], ignore_extra_keys: bool = False) -
         new_instance._created_from_null = self._created_from_null
 
         return new_instance
+
+    def _compare(self, other: object) -> bool:
+        if not isinstance(other, type(self)):
+            return False
+
+        if len(self) != len(other):
+            return False
+
+        if self._created_from_null != other._created_from_null:
+            return False
+
+        items = cast(list[AvdBase], self._items)
+        other_items = cast(list[AvdBase], other._items)
+        return all(item == other_items[index] for index, item in enumerate(items))
```

Comment:
1. are we expecting the lists to be sorted?
2. not sure why we use `==` here and not `_compare` as in indexed list

Reply:
well lists are ordered. So if they are not having the same order, they are not the same.